packages/client/src/forms/index.ts

```diff
@@ -77,9 +77,11 @@ export enum Action {
   REGISTER_APPLICATION = 'register',
   COLLECT_CERTIFICATE = 'collect certificate',
   REJECT_APPLICATION = 'reject',
-  ARCHIVE_DECLARATION = 'archive',
+  ARCHIVE_APPLICATION = 'archive',
```

Comment:
This should have been deleted as we already have `ARCHIVE_DECLARATION` here

Reply:
Removed